pom.xml

```diff
@@ -66,10 +66,10 @@
     <version.org.reflections>0.9.11</version.org.reflections>
     <version.javax.inject>1</version.javax.inject>
 
-    <version.org.infinispan>10.0.0.Beta3</version.org.infinispan>
-    <version.org.infinispan.protostream>4.3.0.Alpha6</version.org.infinispan.protostream>
+    <version.org.infinispan>10.0.0.Beta5</version.org.infinispan>
+    <version.org.infinispan.protostream>4.3.0.Alpha9</version.org.infinispan.protostream>
 
-    <version.io.quarkus>0.22.0</version.io.quarkus>
+    <version.io.quarkus>0.23.1</version.io.quarkus>
     <version.com.oracle.substratevm>19.2.0</version.com.oracle.substratevm>
```

Comment:
should we also align substrate https://github.com/quarkusio/quarkus/blob/0.23.1/bom/runtime/pom.xml#L69

Reply:
done